src/org/zaproxy/zap/extension/params/SiteParameters.java

```diff
@@ -60,7 +61,7 @@ public void setSite(String site) {
 	 * @since 2.5.0
 	 */
 	public boolean hasParams() {
-		return !cookieParams.isEmpty() || !urlParams.isEmpty() || !formParams.isEmpty();
+		return !cookieParams.isEmpty() || !urlParams.isEmpty() || !formParams.isEmpty() || !headerParams.isEmpty();
```

Comment:
The JavaDoc should be updated to also mention the headers (or, remove the type enumeration?).